src/index.js

```diff
@@ -76,18 +76,22 @@ class Masq {
         resolve()
       })
     })
+
+    await this._deleteSessionInfo(false)
   }
 
   async signout () {
     this.userId = null
     this.userAppDb = null
 
     await this.disconnect()
-    this._deleteSessionInfo()
+    this._deleteSessionInfo(true)
   }
 
-  _deleteSessionInfo () {
-    window.localStorage.removeItem('userId')
+  _deleteSessionInfo (deleteLocal) {
+    if (deleteLocal) {
```

Comment:
is it necessary ? from https://developer.mozilla.org/en-US/docs/Web/API/Storage/removeItem `https://developer.mozilla.org/en-US/docs/Web/API/Storage/removeItem`

Reply:
You mean is the condition necessary ? or the function ?

Reply:
condition